dbee/clients/common/client.go

```diff
@@ -19,12 +32,14 @@ func NewClient(db *sql.DB) *Client {
 	}
 }
 
-func (c *Client) Conn() (*Conn, error) {
+func (c *Client) Conn() (DatabaseConnection, error) {
```

Comment:
Tying to the comment above, we want THE Connection, not A connection here.

Same for other places

Reply:
[b9d0035](https://github.com/kndndrj/nvim-dbee/pull/34/commits/b9d0035a1eeda0d6b564928c154cd39e1d7ea90e)